SofascoreProject/SofascoreProject/View/CountryAndNameHolder.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CountryAndNameHolder.swift
+//  SofascoreProject
+//
+//  Created by Sven Leko on 16.03.2024..
+//
+
+import Foundation
+import SofaAcademic
+import UIKit
+
+public class CountryAndNameHolder : BaseView {
+    var country: UILabel = .init()
```

Comment:
nema smisla `country` za ime varijable jer nije tip `Country` nego `UILabel`. Uglavnom uvijek u ime ako je view/label/imageView... dodaj taj suffix.
```suggestion
    var countryNameLabel: UILabel = .init()
```

Reply:
Obavezno svi viewovi u cijeloj aplikaciji moraju biti `private`, zelis enkapsulirat i sakrit internu strukturu od pozivatelja. Ostavis samo javne settere koji postavljaju text, boju, sliku, je li nesto hidden itd.